quickwit/quickwit-config/src/storage_config.rs

```diff
@@ -416,10 +434,18 @@ mod tests {
         storage_configs.apply_flavors();
 
         let gcs_storage_config = storage_configs[0].as_s3().unwrap();
+        assert!(gcs_storage_config.force_path_style_access);
+        assert!(gcs_storage_config.disable_multi_object_delete);
```

Comment:
hum, `gcs_storage_config` as a misleading name, should be digital ocean

Reply:
Good catch!